subprojects/logging/src/main/java/org/gradle/internal/logging/events/RenderableOutputEvent.java

```diff
@@ -16,13 +16,16 @@
 
 package org.gradle.internal.logging.events;
 
+import org.gradle.api.Nullable;
 import org.gradle.api.logging.LogLevel;
 import org.gradle.internal.logging.text.StyledTextOutput;
 
 public abstract class RenderableOutputEvent extends CategorisedOutputEvent {
+    private final OperationIdentifier operationId;
```

Comment:
@adammurdoch I wasn't necessarily suggesting to use the category. I was saying that due to my lack of understanding of the build operation API and category seemed like the information that kind of indicates the type. What I meant is: some information that indicates the _type_ of build operation as strongly-typed field in the interim until we have full support for build operations in the console. As a result `OperationIdentifier` would contain other information and should likely be renamed.